### ISSUE TITLE
Person 2072 has a deprecated record and an active record

### ISSUE THREAD

**Issue by wlpotter** [srophe/srophe-app-data#959]
https://syriaca.org/person/2072 is Sergius of Tella and currently throws an error which I believe is due to its having a record both in the saints directory and in the deprecated directory:

- https://github.com/srophe/srophe-app-data/blob/master/data/persons/tei/saints/tei/2072.xml
- https://github.com/srophe/srophe-app-data/blob/master/data/deprecated/persons/tei/saints/tei/2072.xml

The deprecated record says it was merged into http://syriaca.org/person/1463 but the record for 1463 doesn't mention this merge, and it is missing data from 2072.

As far as I can tell, the two versions of 2072 have the same data, so I think we need to merge 2072 into 1463, mark the deprecated record as merged, and delete the version that is on master.

**Comment by dlschwartz** [comment]
I still have a record 2072. See https://github.com/srophe/srophe-app-data/issues/704

**Comment by wlpotter** [comment]
Yes, it looks like the merge was not fully completed. I can work on this
